org/mozilla/jss/provider/javax/net/JSSContextSpi.java

```
@@ -88,13 +88,23 @@ public SSLSessionContext engineGetServerSessionContext() {
     }
 
     public SSLServerSocketFactory engineGetServerSocketFactory() {
-        logger.debug("JSSContextSpi.engineGetServerSocketFactory() - not implemented");
-        return null;
+        logger.warn("JSSContextSpi.engineGetServerSocketFactory() - not implemented - stubbing with SunJSSE");
+        try {
+            SSLContext jsse = SSLContext.getInstance("TLS", "SunJSSE");
+            return jsse.getServerSocketFactory();
+        } catch (Exception e) {
+            return null;
```

Let's rethrow the exception to help troubleshooting:
```
throw new RuntimeException(e);
```